plotting/timeseries.py

```diff
@@ -110,15 +110,36 @@ def load_data(self):
                 # Under the current API this indicates that velocity data is being
                 # loaded. Save each velocity component (X and Y) for possible CSV
                 # export later.
-                """
-                self.quiver_data = [point_data[:, 0, :], point_data[:, 1, :]]
 
-                point_data = np.ma.expand_dims(
-                    np.sqrt(
-                        point_data[:, 0, :] ** 2 + point_data[:, 1, :] ** 2
-                    ), 1
-                )
-                """
+                velocity_variables = ['vozocrtx', 'vomecrty']
```

Comment:
We actually have other velocity variables like sea ice and surface wind. So the problem turns into how do we find the component variables when given a magnitude variable? Our magnitude variables always start with `mag` so that might be something to consider... This will also affect how the CSV column headers are set below (i.e. we can't have them hard-coded to water velocity)